app/assets/javascripts/spotlight/user/zpr_links.js.erb

```diff
@@ -3,10 +3,10 @@ Spotlight.onLoad(function(){
     $('#ajax-modal .modal-content').html('<div id="osd-modal-container"></div>');
     var controls = [
       '<div class="controls">',
-      '  <a id="osd-zoom-in"><span class="glyphicon glyphicon-plus-sign"></span></a>',
-      '  <a id="osd-zoom-out"><span class="glyphicon glyphicon-minus-sign"></span></a>',
-      '  <a id="osd-home"><span class="glyphicon glyphicon-resize-small"></span></a>',
-      '  <a id="osd-full-page"><span class="glyphicon glyphicon-fullscreen"></span></a>',
+      '  <a id="osd-zoom-in"><%= Blacklight::Icon.new('add_circle').file_source %></a>',
```

Comment:
😢 that we can't get access to the helper here. I have no alternatives to suggest.

Reply:
Yeah, we also need the `file_source` and not the `svg` (which the helper returns) because of the XML declaration (AFAICT).  Although there may have been some sanitization that could have addressed that (but yeah, still no helper access unfortunately).